react/homework.md

```diff
@@ -0,0 +1,135 @@
+# React homework
+
+## Overview
+
+Develop the app with two divs on the page.
+The first div should contain a search bar that allows to make a request to the certain api. The results of the request should be displayed using cards below the search bar.
+The second div should be a form that allows to enter information and display it as cards with records below.
+Also the second tab will be used to display the information from a card that was clicked on.
+
+The homework should be done in one codesandbox. Below are the steps with detailed requirements.
+
+  **The usage of Redux or any other solutions for state management is prohibited.**
+
+  **The usage of any libraries with components or form libraries is prohibited.** 
+
+## Steps
+
+1. Create a new app using the React template on https://codesandbox.io
+2. Use function components with hooks.
+3. Draw the Search Bar and Cards in the div tag with id "search-bar".
+4. Input value should be saved to LocalStorage during component’s unmount. During the initialization pick the value from LocalStorage and show it.
```

Comment:
please add a tick how to work with LocalStorage
probably some YouTube link

Reply:
done